draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
```

Comment:
OK, makes sense. 

Does this also affect senders that have been set to "recvonly" by setDirection? (i.e. if you set recvonly direction and call createOffer, is a=msid still present? If not, and you change the track with replaceTrack and call setDirection("sendrecv"), which MSID is used?)